src/components/Nav/Nav.stories.js

```diff
@@ -1,24 +1,27 @@
 /**
  * @file Navigation stories.
- * @copyright IBM Security 2019
+ * @copyright IBM Security 2019 - 2020
  */
 
-import React from 'react';
+import { action } from '@storybook/addon-actions';
 import { storiesOf } from '@storybook/react';
+import React from 'react';
 
 import { components } from '../../../.storybook';
 import { Nav, NavItem, NavList } from '../..';
 
-storiesOf(components('Nav'), module).add(
-  'default',
+const { name } = Nav;
```

Comment:
@SimonFinney Was this left over from something else, or am I missing context? I'm not sure what it's doing here aside from renaming two (out of several) occurrences of the word "Nav"?

Reply:
That's right! It was a minor refactor while I was in adding the `onClick` handler